packages/wallets/src/routes/Router.tsx

```diff
@@ -28,8 +28,20 @@ export type TRoute = '/endpoint' | `?${string}` | `${TWalletsRoute}`;
 
 // wallets routes which have their states
 interface WalletsRouteState {
-    '/wallet/account-transfer': { shouldSelectDefaultWallet: boolean; toAccountLoginId: string };
-    '/wallet/transactions': { showPending: boolean; transactionType: 'deposit' | 'withdrawal' };
+    '/': { accountsActiveTabIndex: number };
+    '/wallet/account-transfer': {
+        accountsActiveTabIndex: number;
+        shouldSelectDefaultWallet: boolean;
+        toAccountLoginId: string;
+    };
+    '/wallet/deposit': { accountsActiveTabIndex: number };
```

Comment:
Question: Shouldn't `accountsActiveTabIndex` also be added for /wallet/on-ramp?

Reply:
@lubega-deriv 
since you can't reach out onramp from the main page, using CTA we can't pass `accountsActiveTabIndex` to onramp 🤔